include/tvm/runtime/vm/bytecode.h

```diff
@@ -26,6 +26,7 @@
 
 #include <tvm/runtime/data_type.h>
 #include <tvm/support/logging.h>
+#include <string>
```

Comment:
Consider using `tvm/runtime/container.h`'s `class String`, as in the runtime we want to use our TVM's object system like other runtime's components.

Reply:
I think it's just fine since the result of InstructionStrRepr() is not a part of runtime's state, it's just used for profiling.